src/wx/cmdevents.cpp

```diff
@@ -2766,10 +2769,26 @@ EVT_HANDLER_MASK(ChangeFilter, "Change Pixel Filter", CMDEN_NREC_ANY)
     }
 
     wxString msg;
-    msg.Printf(_("Using pixel filter %s"), pixel_filters->GetString(gopts.filter));
+    msg.Printf(_("Using pixel filter:") + " %s", pixel_filters->GetString(gopts.filter));
     systemScreenMessage(msg);
 }
 
+static const wxString&
+ifb_short_name(const wxChoice* ifbs, unsigned num)
+{
+    static std::map<unsigned, wxString> cache;
+
+    if (cache.count(num))
+        return cache[num];
```

Comment:
You end up doing 2 lookups here. Using `find()` is probably what you want, even if it means adding the usual boilerplate to check against `end()`.